packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -3,34 +3,34 @@
 import android.content.Context;
 import android.content.res.Resources;
 import android.graphics.Bitmap;
-import android.graphics.BlurMaskFilter;
 import android.graphics.Canvas;
 import android.graphics.Color;
 import android.graphics.Paint;
 import android.graphics.Path;
 import android.graphics.Rect;
 import android.graphics.RectF;
 
-import androidx.annotation.NonNull;
 import androidx.core.content.res.ResourcesCompat;
 
 public class CozyMarkerBuilder {
-    private final int bubblePointSize;
-    private final Bitmap defaultClusterMarker;
-    private final Paint clusterTextPaint;
-    private final Paint bubbleTextPaint;
+    private final int shadowSize = 3;
```

Comment:
In the way the markers are being drawn this variable is being used to add a few px to the bitmap to make space for the shadow.

Reply:
what about adding this comment to the code itself?